health_check/storage/backends.py

```diff
@@ -1,8 +1,13 @@
 import uuid
 
+import django
 from django.conf import settings
 from django.core.files.base import ContentFile
-from django.core.files.storage import get_storage_class
+
+if django.VERSION >= (4, 2):
```

Comment:
Used the versioned block approach over the try/catch `ImportError` as this is [automatically fixable by `django-upgrade`](https://github.com/adamchainz/django-upgrade#versioned-blocks).